### ISSUE TITLE
Update instructions around exposing from a different ns

### ISSUE THREAD

**Issue by pwright** [skupperproject/skupper-docs#159]
ATM, there's https://skupper.io/docs/cli/index.html#exposing-services-from-different-ns
However, there are now options to :

* use --enable-cluster-permissions in skupper init

* and --target-namespace in skupper expose

Add doc for these options

**Comment by pwright** [comment]
@grs Should we replace existing doc above? or add an optional extra procedure?

Also, does user need extra privs to  --enable-cluster-permissions (eg cluster-admin)

**Comment by grs** [comment]
Yes, --enable-cluster-permissions requires cluster level privileges. I think we should expand the section to show both targeting a service and a set of pods (e.g. deployment) and explain the differences between the two.
